src/main/java/uk/ac/ebi/tsc/portal/api/team/controller/TeamRestController.java

```diff
@@ -272,7 +270,9 @@ public TeamResource getTeamByName(HttpServletRequest request, Principal principa
 
 		List<TeamResource> resourceList = new ArrayList<>();
 		for (Team team: memberTeams){
-			resourceList.add(teamService.setManagerUserNames(new TeamResource(team), token));
+			TeamResource teamResource = teamService.setManagerUserNames(new TeamResource(team), token);
+			teamResource = teamService.populateTeamContactEmails(teamResource, principal);
+			resourceList.add(teamResource);
```

Comment:
I doubt whether we need to add any details to a resource that has just been deleted.

Reply:
yes it would be overkill here too, don't know why manager usernames was set too

Reply:
This iss done